BusBoard.ConsoleApp/CliInterface.cs

```diff
@@ -28,16 +27,24 @@ public static void RunUserCommandLoop(TflApi tflApi)
                 {
                     var predictions = tflApi.GetSortedArrivals(input);
 
-                    foreach (var prediction in predictions.Take(3))
-                    {
-                        Console.WriteLine(prediction.PredictionSummary);
-                    }
+                    Console.WriteLine(SummarisePredictions(predictions));
                 }
                 catch (TflStopNotFoundException e)
                 {
                     Console.WriteLine(e.Message);
                 }
             }
         }
+
+        public static string SummarisePredictions(IEnumerable<Prediction> predictions, int count = 5)
```

Comment:
Any reason for not making this private and removing the count parameter? 🙂

Reply:
Nope, probably left over from debugging... Fixed in 200e09b

Reply:
What about the count parameter? 🙂